src/pconn.h

```diff
@@ -80,7 +80,7 @@ class IdleConnList: public hash_link, private IndependentRunner
     Comm::ConnectionPointer *theList_;
 
     /// Number of entries theList can currently hold without re-allocating (capacity).
-    int capacity_;
+    unsigned int capacity_;
     ///< Number of in-use entries in theList
     int size_;
```

Comment:
Similar to the other change request, do not let the two related data members to grow apart (and use `size_t` for counting items in a single container):

```suggestion
    size_t capacity_;
    ///< Number of in-use entries in theList
    size_t size_;
```

Reply:
Done in [2d232fa](https://github.com/squid-cache/squid/pull/1118/commits/2d232fadbbcc6c65961b6313770cfee113da94ab)